.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2020  SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+name: 'test'
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  test:
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix:
+        os: [ ubuntu, windows, macos ]
+    runs-on: ${{ matrix.os }}-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+
+    - run: git config --global core.autocrlf input
+      shell: bash
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Does git need to be told to enable symlinks on Windows like it does on Travis?

Reply:
I now added it to the previous step.

Reply:
Any idea why this didn't fail on the windows github action previously?

Reply:
My understanding is that symlinks were traditionally not supported on Windows. However, some of the latest versions do support them. My guess is that the Windows version used in GHA is newer than the one in Travis. GHA is Azure, after all.

On the other hand, GHA has been using Git for Windows' bash as the default bash shell on Windows. However, that changed to MSYS2 some days ago, and it's probably going to be reverted, before switching again. See https://github.com/actions/virtual-environments/issues/1525 and this comment from Git for Windows' maintainer in MSYS2's gitter channel: [October 21, 2020 9:43 AM](https://gitter.im/msys2/msys2?at=5f8fe6bd7be0d67d278875d7).

So, I cannot really tell.